cloud_storage_client/storage.py

```diff
@@ -117,8 +117,10 @@ def upload_file(self, src_file, dst_file, total_retries=4, retries_count=0, seco
                 self._upload_file(src_file, dst_file)
             except IncorrectCredentialsException as e:
                 raise e
+            except NotFoundException as e:
+                print('Not found {}'.format(e))
```

Comment:
Shouldn't we raise the exception so it is captured by the app using this lib?

Reply:
Yes, totally right, removed the comment and raise the exception.